input/pagecontent/index.md

```diff
@@ -18,460 +25,98 @@ This IG uses the following conventions:
-  ```
-
-  ```
-  * onsetDateTime = "2019-04-02"
-  ```
-
-  ```
-  * status = #arrived
-  ```
-
-* **Binding rules** are used on elements with coded values to specify the set of enumerated values for that element. Binding rules include [one of FHIR R4's binding strengths](http://hl7.org/fhir/R4/valueset-binding-strength.html) (example, preferred, extensible, or required). For example:
-
-  ```
-  * gender from http://hl7.org/fhir/ValueSet/administrative-gender (required)
-  ```
-
-  ```
-  * address.state from USPSTwoLetterAlphabeticCodes (extensible)
-  ```
-
-* **Cardinality rules** constrain the number of occurrences of an element, either both upper and lower bounds, or just upper or lower bound. For example:
-
-  ```
-  * note 0..0
-  ```
-
-  ```
-  * note 1..
-  ```
-
-  ```
-  * note ..5
-  ```
-
-* **Contains rules** are used for slicing and extensions. Both cases involve specifying the type of elements that can appear in arrays.
-
-  The following rule slices Observation.component into the two components of blood pressure:
-
-  ```
-  * component contains systolicBP 1..1 and diastolicBP 1..1
-  ```
-  Typically, after defining the slices, rules constraining the slices are required. In the blood pressure example, the slices have different codes and the same data type and units of measure:
-
-  ```
-  * component[systolicBP].code = http://loinc.org#8480-6 // Systolic blood pressure
-  * component[systolicBP].value[x] only Quantity
-  * component[systolicBP].valueQuantity = UCUM#mm[Hg]
-  * component[diastolicBP].code = http://loinc.org#8462-4 // Diastolic blood pressure
-  * component[diastolicBP].value[x] only Quantity
-  * component[diastolicBP].valueQuantity = UCUM#mm[Hg]
-  ```
-    The syntax for extensions is very similar. Here are two examples:
-
-  ```
-  * bodySite.extension contains laterality 0..1
-  ```
-
-  ```
-  * extension contains
-      treatmentIntent 0..1 MS and
-      terminationReason 0..* MS
-  ```
-  
-   When incorporating an extension defined by FHIR, an external IG, or using FSH's `Extension` keyword, a modified syntax that assigns local name to the extension is used:
+Because it is a (proposed) HL7 FHIR standard, a variety of tooling can be built around FSH. The most notable tool currently is [SUSHI](https://fshschool.org/docs/sushi/), a reference implementation for transforming FSH into FHIR artifacts that are directly integrated with the [HL7 FHIR Implementation Guide Publishing tool](https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation). SUSHI has a number of features to seamlessly go from FSH to an IG.
 
-  ```
-  // Adding standard FHIR extensions in an AllergyIntolerance profile:
+### Ballot Status
 
-  * extension contains
-      allergyintolerance-certainty named substanceCertainty 0..1 MS and
-      allergyintolerance-resolutionAge named resolutionAge 0..1 MS
-  ```
+This Implementation Guide has been balloted as STU with the intention to go normative in a subsequent ballot cycle.
```

Comment:
Should you spell out STU?  I think probably...

Reply:
did it.